docs/pages/database-access/guides/mysql-self-hosted.mdx

```diff
@@ -1,15 +1,21 @@
 ---
 title: Database Access with Self-Hosted MySQL
-description: How to configure Teleport Database Access with self-hosted MySQL
+description: How to configure Teleport Database Access for self-hosted MySQL databases.
 ---
 
 # Self-Hosted MySQL
 
-## Create Certificate/Key Pair
+This Guide describes how to configure Teleport Database Access for self-hosted MySQL databases.
 
-Teleport uses mutual TLS for authentication to MySQL instances. As such,
-self-hosted MySQL instances must be configured with Teleport's certificate
-authority and a certificate/key pair that Teleport can validate.
+(!docs/pages/includes/permission-warning.mdx!)
+
+(!docs/pages/includes/backup-warning.mdx!)
+
+## Create role and user
+
+(!docs/pages/database-access/includes/db-user-roles.mdx!)
```

Comment:
I found this to be a bit confusing since I was creating a user out of sequence. We'll add this here at the beginning.